portfolio/src/main/webapp/index.html

```diff
@@ -1,42 +1,52 @@
 <!DOCTYPE html>
 <html>
```

Comment:
nit: add a language to the html tag. For example, since we're in the US, you could use:

```html
<html lang="en-US">
```